apm-agent-bootstrap/src/main/java/bootstrap/dispatcher/IndyBootstrapDispatcher.java

```diff
@@ -91,7 +91,11 @@ public static void logAdviceException(Throwable exception) {
                 printStackTrace(exception);
             }
         } catch (Throwable t) {
-            printStackTrace(t);
+            try {
+                printStackTrace(t);
+            } catch (Exception e) {
```

Comment:
```suggestion
            } catch (Throwable e) {
```
`StackOverflowError` is an error not exception, so needs to be `Throwable'

Reply:
Good catch! (pun intended)